pages/index.tsx

```diff
@@ -1,25 +1,65 @@
 import type { NextPage } from "next";
-import React, { useState } from "react";
-import { EuiText, EuiFieldText, EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
+import React, { useState, useEffect } from "react";
+import { EuiText, EuiFieldText, EuiFlexGroup, EuiFlexItem, EuiButton } from "@elastic/eui";
+import { useRouter } from 'next/router';
 
 const Home: NextPage = () => {
-  const [room, setRoom] = useState("");
+  const router = useRouter();
+  const [newRoom, setNewRoom] = useState("");
+  const [joinRoom, setJoinRoom] = useState('');
+
+  useEffect(() => {
+    if(!newRoom) {
+      setNewRoom(roomGen());
+    }
+  }, [newRoom]);
+
+  const roomGen = () => {
+    function a() {
+      return String.fromCharCode(65+Math.floor(Math.random() * 26));
+    }
+    return a().concat(a(),a(),a());
+  }
+
+  const onChange = (e) => {
+      setJoinRoom(e.target.value.replace(/[1-9]/g, "").toUpperCase())
```

Comment:
Obviously fine for now, but should add a ticket or something to update this to letters only.

Reply:
does it not already? I can only type letters.

Reply:
++ to symbols. I can type those and letters